.github/workflows/android.yml

```diff
@@ -126,6 +137,12 @@ jobs:
           submodules: 'true'
           ref: ${{ env.TARGET_BRANCH }}
 
+      - name: Init submodules
```

Comment:
Can we avoid such workaround? why `submodules: true` for `actions/checkout` does not work?

Reply:
@aobolensk fixed